### ISSUE TITLE
Bad characters when writing the top tilde and then the asterisk *˜*

### ISSUE THREAD

**Issue by webaddicto** [notepad-plus-plus/notepad-plus-plus#12053]
### Description of the Issue

When writing asterisk top-tilde asterisk:

`*˜*`

The asterisk on the right (the character after the top-tilde) is shown merged with the top tilde.

Here is a screenshot comparing Notepad++ and Notepad:

![top-tilde-issue](https://user-images.githubusercontent.com/8770136/187250148-e5855843-ed2f-436c-bc06-88604c85b90e.png)

### Steps to Reproduce the Issue

1. Open Notepad++
2. Write `*˜*`
3. See that the top-tilde is joined/mixed with the right asterisk.

### Expected Behavior

It should show the 3 characters:

`*˜*`

### Actual Behavior

It writes the right asterisk over/mixed on the top-tilde.

Here is a screenshot comparing Notepad++ and Notepad:

![top-tilde-issue](https://user-images.githubusercontent.com/8770136/187250148-e5855843-ed2f-436c-bc06-88604c85b90e.png)

### Debug Information

I am using Notepad++ v8.4.4 (Happy Users' Edition) with default plugins (a fresh install) on Windows 10 64-bit.

**Comment by alankilborn** [comment]
Try turning on DirectWrite:

![image](https://user-images.githubusercontent.com/77065706/187254974-f205652e-f41f-4d26-a941-5b9842360f26.png)

**Comment by Chealer** [comment]
@webaddicto: What do you mean by "the top tilde"? Do you mean the combining tilde (the combining character)?

**Comment by webaddicto** [comment]
Yes, basically if you write `*˜*` on a blank tab on Notepad++ I see like 2 chars (seems the tilde is merged with the right asterisk).

On Windows Notepad app it is displayed correctly, all the 3 different chars are visible.

**Comment by alankilborn** [comment]
@webaddicto Did you happen to read this?: https://github.com/notepad-plus-plus/notepad-plus-plus/issues/12053#issuecomment-1230584263

This issue should be closed.

**Comment by Chealer** [comment]
@webaddicto: 
> Yes, basically if you write `*˜*` on a blank tab on Notepad++ I see like 2 chars (seems the tilde is merged with the right asterisk).

You already wrote that. What you did not explain is what you mean by "top tilde". Given your answer, assuming you are indeed talking about U+0303 ◌̃ COMBINING TILDE, that is a combining character, and I see no evidence that it should not be combinable with asterisks, so I challenge the _Expected behavior_ you describe and this report’s validity.

In any case, please give this a more general and clearer title. I suggest:
> U+0303 ◌̃ COMBINING TILDE and following asterisk (˜*) are combined

<details>
  <summary>License</summary>
This comment and any other from myself on this ticket is offered under the terms of CC0 1.0.
</details>
